packages/amplify-cli/src/extensions/amplify-helpers/resource-status.ts

```diff
@@ -9,13 +9,17 @@ import { CLOUD_INITIALIZED, CLOUD_NOT_INITIALIZED, getCloudInitStatus } from './
 import { ServiceName as FunctionServiceName, hashLayerResource } from 'amplify-category-function';
```

Comment:
This file has grown way too huge - if possible can we look at splitting this file up (only if it makes sense to do so )